printer_mods/beaugalbraith/Nova_Printhead/README.md

```diff
@@ -0,0 +1,3 @@
+Printhead for the Nova hotend. Only tested on my 2.4.
+
```

Comment:
Can you include your images in the README?  The syntax is

    ![IMG_7030](images/IMG_7030.jpg)

Reply:
Fixed with commit b7ae011